_layouts/default.html

```diff
@@ -30,7 +30,6 @@
         {{ content }}
       </div>
     </div>
-    {% include footer.html %}
-    {{ page.end_scripts }}
+    {% include footer.html %} {{ page.end_scripts }}
```

Comment:
Now this I would expect to be on separate lines though as it’s completely different things? Fine with either though :)

Reply:
Fixing now.